make-exec.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
+ `set -e` at the beginning
http://stackoverflow.com/questions/3474526/stop-on-first-error